habana/1.17.1/ubi9-python-3.11/Pipfile

```
@@ -0,0 +1,23 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+# JupyterLab packages
+jupyterlab = "~=3.6.7" # Wait on upgrade till plugins are ready
```

we will need to upgrade jupyterlab to 4.2. (This upgrade will affect also the the rest jupyter extensions for upgrade)
```suggestion
jupyterlab = "~=4.2.4" 
```

---

Updated the version accordingly. Thanks!